src/core/trulens/core/feedback/endpoint.py

```diff
@@ -50,6 +56,15 @@
 https://www.trulens.org/component_guides/other/no_context_warning for more information.
 """
 
+_RE_NO_RETRY = re.compile(
+    "("
+    + ("|".join(["authentication", "unauthorized", "expired", "quota"]))
```

Comment:
I'm unsure what error codes are associated with the text checks I did. It might be a more robust approach than looking at the text as is implemented now. Can we make that a todo?